DOCUMENTATION.md

```diff
@@ -498,7 +499,15 @@ response = notifications_client.send_precompiled_letter_notification(
 A unique identifier you create if necessary. This reference identifies a single unique notification or a batch of notifications. It must not contain any personal information such as name or postal address.
 
 ```php
-$reference = 'STRING',
+$reference = 'STRING';
+```
+
+#### postage (optional)
```

Comment:
This should probably go after `$pdf_file` to match the order of the function arguments